--- a/v8/src/base/macros.h
+++ b/v8/src/base/macros.h
@@ -392,21 +392,14 @@
 
 #ifdef V8_OS_WIN
 
-// Setup for Windows shared library export.
-#ifdef BUILDING_V8_SHARED
-#define V8_EXPORT_PRIVATE __declspec(dllexport)
-#elif USING_V8_SHARED
-#define V8_EXPORT_PRIVATE __declspec(dllimport)
-#else
 #define V8_EXPORT_PRIVATE
-#endif  // BUILDING_V8_SHARED
 
 #else  // V8_OS_WIN
 
 // Setup for Linux shared library export.
 #if V8_HAS_ATTRIBUTE_VISIBILITY
 #ifdef BUILDING_V8_SHARED
-#define V8_EXPORT_PRIVATE __attribute__((visibility("default")))
+#define V8_EXPORT_PRIVATE
 #else
 #define V8_EXPORT_PRIVATE
 #endif
