--- a/content/browser/media/capture/desktop_capture_device.cc
+++ b/content/browser/media/capture/desktop_capture_device.cc
@@ -521,7 +521,7 @@
   std::unique_ptr<webrtc::DesktopCapturer> capturer;
   std::unique_ptr<media::VideoCaptureDevice> result;
 
-#if BUILDFLAG(IS_WIN)
+#if 0
   options.set_allow_cropping_window_capturer(true);
   if (base::FeatureList::IsEnabled(features::kWebRtcAllowWgcDesktopCapturer))
     options.set_allow_wgc_capturer(true);
